libs/langchain/langchain/document_loaders/parsers/pdf.py

```diff
@@ -149,3 +149,79 @@ def lazy_parse(self, blob: Blob) -> Iterator[Document]:
                 )
                 for page in doc.pages
             ]
+
+
+class AmazonTextractPDFParser(BaseBlobParser):
+    """Sends PDF files to Amazon Textract and parses them to generate Documents.
+    For parsing multi-page PDFs, they have to reside on S3.
```

Comment:
nit: https://google.github.io/styleguide/pyguide.html#384-classes

```suggestion

    For parsing multi-page PDFs, they have to reside on S3.
```